### DISCUSSION TITLE
How can I create a native library and use the shared value from the native code ?

### DISCUSSION THREAD

**Discussion by Daavidaviid** [software-mansion/react-native-reanimated#5301] [General]
For instance let's say I have a value like this one :

```typescript
const scale: SharedValue<number>;
```

How could I access this value from the native code ?
It would be a custom native library.

I was thinking maybe using `createWorkletRuntime`: 
https://docs.swmansion.com/react-native-reanimated/docs/threading/createWorkletRuntime

And also, I'll try to get inspiration from `react-native-skia` code

Any idea ?

**Comment by Vimal1464** [comment]
can anyone help me with this
https://github.com/software-mansion/react-native-reanimated/issues/6074
